docs/product/support.rst

```diff
@@ -66,42 +66,46 @@ This should be created via a dedicated VM for testing / support activities.
 Chat
 ====
 
-`Olark <https://www.olark.com>`_ is currently used for chat with our customers
-or possible customers.
+`Kayako <https://sftpplus.kayako.com/>`_ is used for chat with our customers
```

Comment:
do we want to promote sftpplus.kayako.com . Are we really using that website?

I think that is ok to just mention Kayako, without any link

Reply:
It's not promoting it since it's in Styleguide, we use that link to login.. or at least I do.  It doesn't appear in search so I think it's ok to mention publicly the link